wagtail/admin/static_src/wagtailadmin/css/normalize.css

```diff
@@ -421,18 +421,7 @@ input {
   line-height: normal;
 }
 
-/**
- * Address inconsistent `text-transform` inheritance for `button` and `select`.
- * All other form control elements do not inherit `text-transform` values.
- * Correct `button` style inheritance in Chrome, Safari 5+, and IE 6+.
- * Correct `select` style inheritance in Firefox 4+ and Opera.
- */
-
 button,
```

Comment:
pretty sure this line can be removed from what I can see

Reply:
sure